src/oscar/apps/dashboard/catalogue/views.py

```diff
@@ -226,6 +226,8 @@ def __init__(self, *args, **kwargs):
                          'image_formset': self.image_formset,
                          'recommended_formset': self.recommendations_formset,
                          'stockrecord_formset': self.stockrecord_formset}
+        self.creating = None
```

Comment:
I think it's better to define these as class variables, for consistency with what is done in the other mixins in this file, but also because it ensures deterministic overriding at instance level.